service.dataplane.Dockerfile

```diff
@@ -21,7 +26,11 @@ RUN if [ "$TARGETARCH" = "amd64" ]; \
     fi
 RUN rustup target add $(eval cat arch)-unknown-linux-musl
 
-COPY . .
+COPY dataplane dataplane 
+COPY tools/udp-test-server tools/udp-test-server
```

Comment:
Unfortunately, we do. If I recall correctly, the cargo complains if the referenced workspace members are missing, even if you're only building a single package.

I don't think this is _that_ big of a deal, however, because it doesn't get copied over to the second stage.

If you know a work around, please share.